### PR TITLE
GameDB: Some NTSC-K fixes

### PR DESCRIPTION
**Pull request by icup321**
### Description of Changes
Just adds a few fixes for NTSC-K versions of games that were overlooked.

### Rationale behind Changes
Less glitches, less problems.

### Suggested Testing Steps
Make sure patches work.

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -6551,8 +6551,11 @@ SCKA-20029:
   gsHWFixes:
     getSkipCount: "GSC_PolyphonyDigitalGames" # Fixes post processing.
 SCKA-20030:
-  name: "Sly Cooper - Jeonseolui Bibeobseoreul Chajaseo"
+  name: "Sly Cooper and the Thievius Raccoonus [PlayStation 2 Big Hit Series]"
```
Comment (Mrlinkwii) [inline bin/resources/GameIndex.yaml:6554]: use  `name-en 
` so it has both

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -29301,6 +29306,15 @@ SLKA-25459:
   compat: 5
   gsHWFixes:
     cpuSpriteRenderBW: 2 # Fixes textures.
+    halfPixelOffset: 2 # Fixes misaligned bloom.
+    autoFlush: 2 # Fixes soft shadows.
+  patches:
```
Comment (Mrlinkwii) [inline bin/resources/GameIndex.yaml:29311]: did you verify that this patch works this version?

Comment (icup321) [reply bin/resources/GameIndex.yaml:29311]: I think it does, I couldn't tell any difference between the NTSC-U and NTSC-K versions when playing them, memory addresses are also the same.

Comment (Mrlinkwii) [reply bin/resources/GameIndex.yaml:29311]: i would prefer if that could get tested :)

**Comment by refractionpcsx2** [comment]
Can you fix the commit name please, it needs `GameDB:` prefixing to it, thanks :)

**Comment by icup321** [comment]
> Can you fix the commit description please, it needs `GameDB:` prefixing to it, thanks :)

Done.

**Comment by refractionpcsx2** [comment]
Thanks :)

**Pull request merged**